zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -10871,14 +10871,15 @@ components:
         $like:
           $ref: "#/components/schemas/LikeFilterProperty"
     BatchOperationStateEnum:
-      description: The state, one of ACTIVE, COMPLETED, TERMINATED.
+      description: The state, one of CREATED, ACTIVE, SUSPENDED, CANCELED, COMPLETED, PARTIALLY_COMPLETED, FAILED.
```

Comment:
🔧 The enum expresses the possible states already, not needed in the description again.